gff3tool/lib/gff3_merge/auto_replace_tag.py

```diff
@@ -10,42 +10,41 @@
 import re
 import logging
 import subprocess
+import os
 try:
     from subprocess import DEVNULL # py3k
 except ImportError:
-    import os
     DEVNULL = open(os.devnull, 'wb')
 # try to import from project first
-import os
-from os.path import dirname
-if dirname(__file__) == '':
-    lib_path = '../'
-else:
-    lib_path = dirname(__file__) + '/../'
-sys.path.insert(1, lib_path)
-from gff3 import Gff3
-import gff3_to_fasta
+lib_path = os.path.dirname((os.path.dirname(os.path.abspath(__file__))))
+from gff3tool.lib.gff3 import Gff3
+import gff3tool.bin.gff3_to_fasta as gff3_to_fasta
+import shutil
 
-__version__ = '0.0.3'
+from gff3tool.bin import version
+
+__version__ = version.__version__
 
 
 def main(gff1, gff2, fasta, outdir, scode, logger, all_assign=False, user_defined1=None, user_defined2=None):
     logger_null = logging.getLogger(__name__+'null')
     null_handler = logging.NullHandler()
     logger_null.addHandler(null_handler)
     if not os.path.isdir(outdir):
-        subprocess.Popen(['mkdir', outdir]).wait()
+        os.makedirs(outdir)
 
     tmpdir = '{0:s}/{1:s}'.format(outdir, 'tmp')
     if not os.path.isdir(tmpdir):
-        subprocess.Popen(['mkdir', tmpdir]).wait()
+        os.makedirs(tmpdir)
 
     #Check if there is a non-coding transcript
     transcripts = set()
     transcripts_type = set()
     gff3_1 = Gff3(gff_file=gff1, fasta_external=fasta, logger=logger)
     gff3_2 = Gff3(gff_file=gff2, fasta_external=fasta, logger=logger)
 
+    makeblastdb_path = os.path.join(lib_path, 'ncbi-blast+/bin/makeblastdb')
+    blastn_path = os.path.join(lib_path, 'ncbi-blast+/bin/blastn')
```

Comment:
Use syntax like `os.path.join(lib_path, 'ncbi-blast+', 'bin', 'makeblastdb')`
Original syntax will generate path like `\\some\\path\\ncbi-blast+/bin/blastn` on some Windows systems and failed to be executed on them.